### PR TITLE
feat(deps): updates and adds support for rollup v3

### PR DESCRIPTION
**Pull request by geotrev**
I think this should be a somewhat simple PR but let me know if there are additional things I should test for... the existing plugins seem to hold up.

Addresses #24

### CODE REVIEW

**Review by brody2consult — CHANGES_REQUESTED** [review]
My apologies for the delay, I think we need to add CI testing with Rollup v2 as well.

Maybe the best thing would be to check the output sizes less precisely with Rollup v2. To do this in CI, I would recommend an approach like this:

- add CI task that adds specific npm or Yarn installation of Rollup v2, test with an environmental variable defined if using Rollup v2
- then test some sizes less precisely or consider other adjustments in case of Rollup v2

As an alternative: we *could* also drop support for Rollup v2 in a new 0.x release of this Rollup plugin. I think this could be an acceptable alternative if we document it somewhere as this fork does not seem to have too many users.

Thanks for your contribution, definitely much appreciated!

**Comment by geotrev** [comment]
@brodybits thanks for taking a look! 

Do you anticipate new/changing features that would make backwards compatibility on Rollup v2 beneficial? If not, bumping to a new minor and dropping v2 support works for me. In that case, I can update the README.

Give it some thought, I'm cool making changes in either direction. 👍🏻

**Comment by brody2consult** [comment]
My first choice would be to support both Rollup v2 & v3 with testing, raised #26 to track this.

I can merge this PR for a new minor release if we remove the Rollup v2 option, and resolve #26 whenever someone has time to work on this.

Thanks!

File: package.json [hunk 1]
```diff
@@ -54,11 +54,11 @@
     "lint-staged": "^10.5.4",
     "prettier": "~2.8.3",
     "redux": "^4.2.0",
-    "rollup": "~2.40.0",
+    "rollup": "~3.28.1",
     "strip-ansi": "^6.0.1"
   },
   "peerDependencies": {
-    "rollup": "^2.0.0"
+    "rollup": "^2.0.0 || ^3.0.0"
```
Comment (brody2consult) [inline package.json]: I would like to drop rollup v2 support for now.

At this point test suite works with more recent rollup v2 such as 2.40 & 2.79.1 but not all the way back to `2.0.0`. And the proposed updates will succeed with Rollup pre-release `4.0.0-13`. So I would likely favor moving forward supporting the newer Node.js versions.

```suggestion
    "rollup": "^3.0.0"
```

**Comment by geotrev** [comment]
Sounds good, thanks.

**Review by brody2consult — APPROVED** [review]
Approved & merging with removal of Rollup v2 support, as already discussed, thanks!

**Pull request merged**